### DISCUSSION TITLE
How to connect to headless Debian guest

### DISCUSSION THREAD

**Discussion by tonytx05** [utmapp/UTM#5729] [UTM Help (macOS)]
I followed the instructions [here](https://docs.getutm.app/advanced/headless/) for setting up a headless system. The serial device is created at /dev/ttys000. When I run the command `screen /dev/ttys000`, the terminal seems to open something, but is just blank. Any help on what I'm doing wrong? Is there some configuration necessary on the guest OS that I've missed?

**Comment by jctull** [comment]
I came here looking for an answer to the same issue. Unfortunately I have not found one yet.
